views/plugin/featureaction/featuresoundactionconfig.html

```diff
@@ -0,0 +1,21 @@
+<div class="feature-action-sound-config">
```

Comment:
HTML files should use 2 spaces to indent a new DOM level, 4 spaces to indent multi-line elements.